src/mono/nuget/Microsoft.NET.Sdk.WebAssembly.Pack/build/Microsoft.NET.Sdk.WebAssembly.Browser.targets

```diff
@@ -190,11 +190,15 @@ Copyright (c) .NET Foundation. All rights reserved.
   <Target Name="_ResolveWasmOutputs" DependsOnTargets="ResolveReferences;PrepareResourceNames;ComputeIntermediateSatelliteAssemblies;_ResolveWasmConfiguration;_WasmNativeForBuild">
     <ItemGroup>
       <_WasmConfigFileCandidates Include="@(StaticWebAsset)" Condition="'%(SourceType)' == 'Discovered'" />
+      <_DotNetJsItem Include="@(ReferenceCopyLocalPaths)" Condition="('%(FileName)' == 'dotnet' or '%(FileName)' == 'dotnet.native') and '%(Extension)' == '.js'" />
 
       <!-- Remove dotnet.js/wasm from runtime pack, in favor of the relinked ones in @(WasmNativeAsset) -->
       <ReferenceCopyLocalPaths Remove="@(ReferenceCopyLocalPaths)"
                                Condition="@(WasmNativeAsset->Count()) > 0 and ( '%(FileName)' == 'dotnet' or '%(FileName)' == 'dotnet.native' ) and ('%(Extension)' == '.wasm' or '%(Extension)' == '.js')" />
     </ItemGroup>
+    <PropertyGroup>
+      <_DotNetJsBuildVersion>%(_DotNetJsItem.NuGetPackageVersion)</_DotNetJsBuildVersion>
```

Comment:
Is this meant to be the runtime pack version?

Reply:
Yes

Reply:
This seems fragile. We should have:
```
                         ResolvedRuntimePack=
                             Microsoft.NETCore.App.Runtime.Mono.browser-wasm
                                     FrameworkName=Microsoft.NETCore.App
                                     IsTrimmable=
                                     NuGetPackageId=Microsoft.NETCore.App.Runtime.Mono.browser-wasm
                                     NuGetPackageVersion=8.0.0-dev
                                     PackageDirectory=/workspaces/runtime/artifacts/bin/dotnet-latest/packs/Microsoft.NETCore.App.Runtime.Mono.browser-wasm/8.0.0-dev
                                     RuntimeIdentifier=browser-wasm
```

.. so the version can be extracted from `@(ResolvedRuntimePack)`, of which there should be only 1.